server/components/hyperdrive/index.js

```diff
@@ -3,12 +3,12 @@ const relative = require('relative-date')
 const pretty = require('pretty-bytes')
 const html = require('choo/html')
 
-module.exports = function (root, opts, onclick) {
+module.exports = function (root, entries, send) {
```

Comment:
the `send` aka `onclick` here doesn't do anything server-side. how about just dropping that third arg so as not to confuse future devs?